src/Microsoft.Azure.WebJobs/ParameterBindingData.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.Azure.WebJobs
+{
+    /// <summary>
+    /// Represents data a parameter binding provides that can be used by a function to perform the bind operation to the target itself.
+    /// </summary>
+    sealed public class ParameterBindingData
```

Comment:
This should be ordered "public sealed". It might be generating a warning?

Reply:
done!